beta/examples/tensorflow/classification/main.py

```diff
@@ -181,13 +180,14 @@ def run(config):
 
             compress_model.summary()
 
+            checkpoint = tf.train.Checkpoint(model=compress_model, compression_ctrl=compression_ctrl)
+            checkpoint_manager = tf.train.CheckpointManager(checkpoint, config.checkpoint_save_dir, max_to_keep=None)
```

Comment:
I suggest to provide the user `nncf.tensorflow.callbacks.CheckpointManager(checkpoint, ckpt_dir, save_checkpoint_freq=None)` instead of explicitly using CheckpointManager in the sample.

Reply:
Please check  https://github.com/openvinotoolkit/nncf/pull/679/files#diff-ad5e5a409c18ceb39c3149be02ddcbcb0b8c661fdfc49b44c4f926d0f1b9239c